metadata/contributors.yaml

```diff
@@ -0,0 +1,171 @@
+# List of contributors' contact details
+#
+# reference maintainers/contributers by their id in tutorial metadata files
+#
+bebatut:
```

Comment:
We could maybe order by alphabetic order of the GitHub username. It can be done in a next PR...
Do you think we could also have a script at some point which take all contributors of the repo and check if all are there and if not add them?

Reply:
I don't mind reordering by alphabet in this PR, think this is nice to do, will add this soon. 

And yes, think such a script would be nice. And/or perhaps add a [PR template](https://github.com/blog/2111-issue-and-pull-request-templates) with a checkbox to indicate all contributors have been listed to remind people to think about this when submitting their contributions?